core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1630,7 +1630,7 @@ private long loadDirectoryMetadata(AlluxioURI path, LoadMetadataOptions options)
       throws IOException, InvalidPathException, AccessControlException, FileDoesNotExistException {
     if (mInodeTree.inodePathExists(path)) {
       Inode inode = mInodeTree.getInodeByPath(path);
-      if (inode.isPersisted()) {
+      if (inode.isPersisted() || path.isRoot()) {
```

Comment:
in other words, I don't think it is possible to start Alluxio without root mounted to a UFS